configs/components/recom/recom.yaml.orig

```diff
@@ -0,0 +1,164 @@
+                        numofCO2cycles: "${numofCO2cycles}"
+                        currentCO2cycle: "${currentCO2cycle}"
+                        REcoM_PI: "${REcoM_PI}"
+                paco2_flux_param:
+                        CO2_for_spinup: "${CO2_for_spinup}"
+
+
+choose_lresume:
+        true:
+                add_namelist_changes:
+                        namelist.recom:
+                                pavariables:
+                                        recom_restart: true
+        false:
+                add_namelist_changes:
+                        namelist.recom:
+                                pavariables:
+                                        recom_restart: false
+
+config_files:
+        recom:  recom
+
+config_sources:
+        recom:  "${namelist_dir}/namelist.recom"
+#        io.recom.ciso: "${namelist_dir}/namelist.io.recom.ciso"
+
+restart_name: ""
+
+restart_in_files:
+        bio_restart: bio_restart
+
+restart_in_in_work:
+        bio_restart: fesom.${parent_date!syear}.bio.${restart_name}nc
+
+restart_in_sources:
+        bio_restart: fesom.${parent_date!syear}.bio.nc
+
+restart_out_files:
+        bio_restart: bio_restart
+
+restart_out_in_work:
+        bio_restart: fesom.${end_date!syear}.bio.${restart_name}nc
+
+restart_out_sources:
+        bio_restart: fesom.${end_date!syear}.bio.nc
+
+yearly_outputs: []
+
+monthly_outputs: [diags3d01,diags3d02,CO2y,aCO2,rDIN,rDON,rDOC,rDSi,PAR,NPPn,NPPd,GPPn,GPPd,NNAn,NNAd,GNAn,GNAd,benCalc,benSi,benC,benN,denb,aN,aFe,Hp,CO2f,dpCO2s,pCO2s,bgc01,bgc02,bgc03,bgc04,bgc05,bgc06,bgc07,bgc08,bgc09,bgc10,bgc11,bgc12,bgc13,bgc14,bgc15,bgc16,bgc17,bgc18,bgc19,bgc20,bgc21,bgc22,slp,age01,Kw,alphaCO2,O2f]
+
+daily_outputs: []
+
+12_step_outputs: []
+
+outdata_files:
+        "[[yearly_outputs-->OUTPUT]]": OUTPUT
+        "[[monthly_outputs-->OUTPUT]]": OUTPUT
+        "[[daily_outputs-->OUTPUT]]": OUTPUT
+outdata_in_work:
+        "[[yearly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
+        "[[monthly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
+        "[[daily_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
+outdata_sources:
+        "[[yearly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
+        "[[monthly_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
+        "[[daily_outputs-->OUTPUT]]": OUTPUT_fesom_${start_date!syear!smonth!sday}.nc
+
+choose_computer.name:
+        blogin:
+                add_compiletime_environment_changes:
+                        add_export_vars:
+                                - "NETCDF_DIR=/sw/dataformats/netcdf/intel.18/4.7.3/skl/"
+                                - "LD_LIBRARY_PATH=$NETCDF_DIR/lib/:$LD_LIBRARY_PATH"
+                                - "NETCDF_CXX_INCLUDE_DIRECTORIES=$NETCDF_DIR/include"
+                                - "NETCDF_CXX_LIBRARIES=$NETCDF_DIR/lib"
+
```

Comment:
I recommend removing this whole file. The original files is kept in previous versions of ESM-Tools and we can come to it later.